src/model.js

```diff
@@ -424,11 +434,24 @@ const BookshelfModel = ModelBase.extend({
     let columnNames, candidates;
     if (_.isArray(arguments[1])) {
       columnNames = arguments[1];
-      candidates = _.drop(arguments, 2);
     } else {
       columnNames = null;
-      candidates = _.drop(arguments);
     }
```

Comment:
consider this one without `registry` plugin:
```js
morphTo('imageable', [Site, 'cover_photo'], [Author, 'profile_pic'])
// columnNames: [Site, 'cover_photo']
// morphValues: [Author, 'profile_pic']
```
Actually, yes, it doesn't reflect to old API. I am sorry. But it would make new API more fail proof.